### ISSUE TITLE
VS preview not working on .NET Standard 2.0

### ISSUE THREAD

**Issue by ejongejans** [picoe/Eto#1552]
## Expected Behavior
Working code preview on MainForm.eto.cs and similar files.  

## Actual Behavior
A list of exceptions about double references is thrown. No preview but red screen. (attached)
The application builds/runs fine. It seems to be somehow related to the previewer? 

## Steps to Reproduce the Problem

  1. Start Eto.Forms project with 3 default platforms (gtk, mac, wpf). Defaults to .net standard 1.6
  2. Runs fine; code preview works for .eto.cs files in split window as expected
  3. Change to .net standard 2.0 (reason is i wanted to use [Serializable] attributes. Then code view doesn't work anymore with exceptions as in picture. 

![image](https://user-images.githubusercontent.com/2496269/72262001-4b603c80-3616-11ea-88b5-78e93e7a7142.png)

  - Version: 2.4.1 eto, Extension 2.4.1.1.
  - Platform(s): main app. (platform independent)
  - Operating System(s): windows 10, VS2017, eto extension installed

**Comment by ejongejans** [comment]
Update/more info: 

After a bit of trial and error still no luck. 

If the main app project (that has the .eto.cs files) is changed to a normal windows class library .net 4.6.1 there are only 2 double references. (screenshot) 


![image](https://user-images.githubusercontent.com/2496269/72323310-b22e3600-36a8-11ea-911d-59e69bf25654.png)

**Comment by cwensley** [comment]
Hi @ejongejans, thanks for reporting the issue!

I'm currently working on trying to get the VS extension updated to VS 2019, and won't be able to support the VS 2017 extension at the same time.  However, I will work to get this resolved for the VS 2019 extension.

**Comment by ejongejans** [comment]
Hi, 

I gave it a shot today but it still shows the reference errors with .eto.cs files. 
Screenshots below. 

What i did: 
- update VS2019 to latest version 
- installed latest Eto addin (downloaded from DevOps, 2.4.9999.0)
- started new Eto project (.NET 1.6/2.0) and checked existing project (.NET 4.6.1)
- both have difference references errors in the code preview but build is fine.

Not sure what to do next. The code preview is extremely handy when you are porting a windows forms app and have lots of .resx strings and UI that is set in the code behind of the winform original.. 


.NET Standard 2.0/1.6: 
![image](https://user-images.githubusercontent.com/2496269/74103383-f0a1fe00-4b4b-11ea-965b-290544688c16.png)


.NET Framework 4.6.1: 

![image](https://user-images.githubusercontent.com/2496269/74103454-5ee6c080-4b4c-11ea-8095-99832d8202c8.png)

**Comment by ejongejans** [comment]
Update: 
I've started debugging the Eto.Addin.VisualStudio project. 

Somehow all projects I open when debugging the extension .eto.cs editor from within the debugged visual studio the codepreview works fine. 
When i run a 'release' version of the extension then i still get the above errors. 

I've checked the method 'GetReferences' but in Debug mode that gets 0 references to inject in the editor. 

Is there a way to see which references are injected here in the released version? Perhaps somehow log them ? 


![image](https://user-images.githubusercontent.com/2496269/74108122-4049ef00-4b77-11ea-9f60-45c9464dc03c.png)
